### ISSUE TITLE
Routeless th08 main-game replayss are given medals

### ISSUE THREAD

**Issue by jgdhs27** [n-rook/thscoreboard#408]
TH08 replays that do not have a route (due to getting a game-over before stage 6) are ranked and given medals separately from 6A and 6B replays. Note that th08 Ex replays also do not have a route but should receive medals.

In general, game-overs need to be able to get medals, as in some games (th12, th06), game-overs really could be top 3 scores.

**Comment by n-rook** [comment]
I agree with inaki (https://discord.com/channels/928695122558857216/1051715136500539453/1110346552012198049): ideally, replays without a route should be included and ranked in both scoreboards.
